src/main/webapp/chat.css

```diff
@@ -0,0 +1,34 @@
+#chat {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
```

Comment:
Can you alphabetize declarations? See section 4.2.1 https://google.github.io/styleguide/htmlcssguide.html